lib/customer-deposit-wallet/test/unit/Cardano/Wallet/Deposit/HTTP/OpenAPISpec.hs

```diff
@@ -0,0 +1,55 @@
+module Cardano.Wallet.Deposit.HTTP.OpenAPISpec
+    ( spec
+    ) where
+
+import Prelude
+
+import Cardano.Wallet.Deposit.HTTP.Types.OpenAPI
+    ( generateOpenapi3
+    )
+import Paths_customer_deposit_wallet
+    ( getDataDir
+    , getDataFileName
+    )
+import Test.Hspec
+    ( Spec
+    , describe
+    , it
+    , shouldReturn
+    )
+import Test.Hspec.Golden
+    ( Golden (..)
+    )
+import UnliftIO.Directory
+    ( doesDirectoryExist
+    , doesFileExist
+    )
+
+import qualified Data.ByteString.Lazy.Char8 as BL
+
+spec :: Spec
+spec = do
+    describe "data dir" $ do
+        it "should exist" $ do
+            f <- getDataDir
+            doesDirectoryExist f `shouldReturn` True
```

Comment:
As argued in another comment, we can afford the occasional call to `liftIO`. The function `shouldReturn` expects `IO`, so we don't gain any benefit from `unliftio` here. 🤓

Reply:
fair enough :+1: